Packs/TrendMicroVisionOne/IncidentFields/incidentfield-Trend_Micro_Vision_One_XDR_Created_Time.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Trend Micro Vision One XDR Incident"
+    ],
+    "caseInsensitive": true,
+    "cliName": "trendmicrovisiononexdrcreatedtime",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_trendmicrovisiononexdrcreatedtime",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Trend Micro Vision One XDR Created Time",
```

Comment:
@mikedgibson It is highly recommended to use existing fields before creating new ones in order to avoid duplicates.
The existing field "Occurred" seems to be able to cover this purpose no?

Reply:
Thanks for the guidance. We removed any new incident fields where it made more sense to use an existing field and mapped to the existing fields. Also updated our custom layout to use the existing fields instead of our custom fields.